tools/ci_build/build.py

```diff
@@ -1095,9 +1095,15 @@ def run_android_tests(args, source_dir, config, cwd):
 
 
 def run_ios_tests(args, source_dir, config, cwd):
-    run_subprocess(["xcodebuild", "test", "-project", "./onnxruntime.xcodeproj",
-                    "-scheme",  "onnxruntime_test_all_xc",
-                    "-destination", "platform=iOS Simulator,OS=latest,name=iPhone SE (2nd generation)"], cwd=cwd)
+    cpr = run_subprocess(["xcodebuild", "test", "-project", "./onnxruntime.xcodeproj",
+                          "-configuration", "RelWithDebInfo",
+                          "-scheme",  "onnxruntime_test_all_xc",
```

Comment:
Why is RelWithDebInfo hardcoded? Should this use the 'config' parameter instead?